### ISSUE TITLE
Sandy 2-1 Rescue Drill

### ISSUE THREAD

**Issue by ibanez802** [LegionDark/Issues#651]
**Date & Time**: 3/1/2016
**Frequency**: always
**FFXI Client Version**: 30151223_0 
**Character Name**: Arielle
**Race**: Mithra
**Nation**: Sandioria
**Main Job**: Blue Mage
**Sub Job**: Red Mage
**Area & Coordinates**: La Theine Plateau X:-481 Y:-7 Z:-221 (73)
**Party or Solo**: Solo
**NPC or Monster Name**: Galaihaurat
**ffxiah.com link (for inventory items only)**: 
**Multi-boxing? (multiple clients on same connection)**: No
**Steps To Reproduce The Issue**: Head to La Theine Plateau and speak to Galaihaurat next to a low wall at the bottom-middle box of (E-6) to start the rescue drill exercise. Also spoke with all other NPCS involved with this mission and nothing will work to actually start mission, and the quest A timely Visit is not active.

**Comment by ibanez802** [comment]
Need Sandiorian Rescue Certificate KI to continue.

**Comment by PikaFox** [comment]
The certificate is awarded at the end of the mission iirc.

Chances are, you missed an NPC to talk to. Talk to EVERYONE on the way down.

**Comment by ibanez802** [comment]
I did, like 5 different times.

**Comment by ibanez802** [comment]
They all say,  there is a rescue drill in process, stay clear.

**Comment by TeoTwawki** [comment]
There is sometimes a situation where you must speak to one twice in a row and if you speak to another in between, it resets. It is also possible that the NPC you need isn't the correct name/position because the IDs mismatch between your client and the server. The coming update will fix that. I've been over this mission repeatedly with a client that matches the server and been able to complete it so its not a code error.

**Comment by TeoTwawki** [comment]
Leaving this open for now, if player remains stuck I will personally walk them through and if their client is preventing completion I will fix the variable.

**Comment by ibanez802** [comment]
Okay i tried a few different ways, i will try again

**Comment by ibanez802** [comment]
There was a type in the client version. If this is not the current version i should run update?

**Comment by TeoTwawki** [comment]
No, server is not up to date yet. You might be worse off after updating than before.

**Comment by ibanez802** [comment]
Yeah i tried all different ways and combinations and tryed talking to all NPC's listed on the wiki for this mission, and still no luck.
